tasks/tests/unit/test_test_results_finisher.py

```diff
@@ -351,7 +351,7 @@ def test_upload_finisher_task_call(
         assert expected_result == result
         mock_repo_provider_comments.post_comment.assert_called_with(
             pull.pullid,
-            "### :x: 4 Tests Failed:\n| Tests completed | Failed | Passed | Skipped |\n|---|---|---|---|\n| 4 | 4 | 0 | 0 |\n<details><summary>View the top 3 failed tests by shortest run time</summary>\n\n> <pre>\n> test_name1\n> </pre>\n> <details><summary>Stack Traces | 2s run time</summary>\n> \n> > <pre>Shared failure message</pre>\n> > [View](https://example.com/build_url_1) the CI Build\n> \n> </details>\n\n\n> <pre>\n> Other Class Name\x1ftest_name2\n> </pre>\n> <details><summary>Stack Traces | 3s run time</summary>\n> \n> > <pre>Shared failure message</pre>\n> > [View](https://example.com/build_url_2) the CI Build\n> \n> </details>\n\n\n> <pre>\n> Class Name\x1ftest_name0\n> </pre>\n> <details><summary>Stack Traces | 4s run time</summary>\n> \n> > <pre>&lt;pre&gt;Fourth <br><br>&lt;/pre&gt; | test  | instance |</pre>\n> > [View](https://example.com/build_url_3) the CI Build\n> \n> </details>\n\n</details>\n\nTo view individual test run time comparison to the main branch, go to the [Test Analytics Dashboard](https://app.codecov.io/gh/joseph-sentry/codecov-demo/tests/main)",
+            "### :x: 4 Tests Failed:\n| Tests completed | Failed | Passed | Skipped |\n|---|---|---|---|\n| 4 | 4 | 0 | 0 |\n<details><summary>View the top 3 failed tests by shortest run time</summary>\n\n> ```\n> test_name1\n> ```\n> <details><summary>Stack Traces | 2s run time</summary>\n> \n> > ```\n> > Shared \n> > \n> > \n> > \n> >  &lt;pre&gt; \n> >  \n> > \n> >  | test | test | test &lt;/pre&gt;failure message\n> > ```\n> > [View](https://example.com/build_url_1) the CI Build\n> \n> </details>\n\n\n> ```\n> Other Class Name test_name2\n> ```\n> <details><summary>Stack Traces | 3s run time</summary>\n> \n> > ```\n> > Shared \n> > \n> > \n> > \n> >  &lt;pre&gt; \n> >  \n> > \n> >  | test | test | test &lt;/pre&gt;failure message\n> > ```\n> > [View](https://example.com/build_url_2) the CI Build\n> \n> </details>\n\n\n> ```\n> Class Name test_name0\n> ```\n> <details><summary>Stack Traces | 4s run time</summary>\n> \n> > ```\n> > &lt;pre&gt;Fourth \n> > \n> > &lt;/pre&gt; | test  | instance |\n> > ```\n> > [View](https://example.com/build_url_3) the CI Build\n> \n> </details>\n\n</details>\n\nTo view individual test run time comparison to the main branch, go to the [Test Analytics Dashboard](https://app.codecov.io/gh/test-username/test-repo-name/tests/main)",
```

Comment:
these snapshots are pretty impossible to review. can you format them as multiline strings?